sonarts-core/src/tools/new-rule.ts

```diff
@@ -0,0 +1,271 @@
+import * as fs from "fs";
+import * as stringify from "json-stable-stringify";
+
+const rootFolder = `${__dirname}/../../..`;
+const rspecRuleFolder = `${rootFolder}/sonarts-sq-plugin/sonar-typescript-plugin/src/main/resources/org/sonar/l10n/typescript/rules/typescript`;
+
+if (process.argv.length != 4) {

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: != should be !== ([typescript:S1440](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=michalb%2Fnew-rules-script&issues=AWC8Oukl_Q-0dz4JAzUb&open=AWC8Oukl_Q-0dz4JAzUb)

[ISSUE_KEY]: AWC8Oukl_Q-0dz4JAzUb